src/main.ts

```diff
@@ -4,7 +4,7 @@ import 'regenerator-runtime/runtime' // to use transpiled generator functions
 
 // Vue Libraries
 import Vue from 'vue'
-import vuetify from '@/plugins/vuetify'
+import Vuetify from 'vuetify/lib'
```

Comment:
interesting, what is the difference here?

Reply:
We were calling a plugin that did very little so I moved its code here. I think it's cleaner :)

Reply:
That you can't tell the difference proves the plugin did very little :)